packages/node-app/src/utils/ta.utils.ts

```diff
@@ -57,8 +57,6 @@ const getPersonById = async (
   id: number,
 ): Promise<TAPerson> => {
   const multiSite = session.siteUrl;
-  const queryParams = archived ? `?archived=${archived}` : '';
```

Comment:
This is being used to avoid getting a person that is no longer active, instead of manually checking the archived property

Reply:
This function had the `queryParams` defined but is not using it, you're probably thinking about the function on top of this one, `getPeople`

Reply:
You're right, here we have sure that we are using a person ID that is not archived. Good point